Makefile

```diff
@@ -18,7 +18,8 @@ endif
 # Tools + Environment
 IN_ENV = if [ -e env/bin/activate ]; then . env/bin/activate; fi; source utils/environment.python.sh;
 env:
-	python3 -mvenv env
+	python3.8 -mvenv env --without-pip
+	$(IN_ENV) curl https://bootstrap.pypa.io/get-pip.py | python
```

Comment:
Why this change?  This seems wrong? Shouldn't it be:
```
python3 -mvenv env
python3 -mpip install --upgrade pip
```
?

Reply:
Perhaps your mentioned fix would work better, but this was just a workaround that we implemented to get the "make env" command to run properly. We were running into issues with the python version and module version incompatibilities in the requirements.txt file

Reply:
Please use @litghost's method.

Reply:
I remember now that we were having issues with the Python version because our machine is running Ubuntu 16.04 which has a python3 version of 3.5. I had to use "python3.8" in order for it to work. I may look into this further in order to see if the method mentioned by @litghost would work for me.